openapi/src/parts/tps.yaml

```diff
@@ -0,0 +1,231 @@
+# Terra Policy Service API
+# This API is exposed by the WSM component, so its API definition has to be built in with WSM.
+paths:
+  # Region policy paths
+  /api/policies/v1alpha1/region-constraint:
```

Comment:
For endpoints we're exposing on WSM but also calling from WSM (likely this one), what's the right pattern for using these? 
Is it "call `TpsService`" which is internal for now, but later on will be a wrapper around a client library, or treat this as an external service now and make calls to the controller?

Reply:
There is a piece of the design that is not implemented yet.
I plan to split the controller into:
-  TpsApiController - just pulls the token out of the request and calls
- TpsApiDispatch - that converts between API and internal TPS and calls TpsService

WSM will call TpsApiDispatch using the API objects. If/when we factor TPS out, we rewrite TpsApiDispatch to call the TPS client instead. The rest of WSM should remain unchanged.

I was going to do that on a second pass, but maybe I should do it in this PR. wdyt?